Privacy.md

```diff
@@ -0,0 +1,85 @@
+
+Other third-party tools used are;
+
+1. [OpenStreetMaps](https://openstreetmap.org/). This service goes into operation automatically as soon as the map opens. Read the company's [Privacy Policy](https://wiki.osmfoundation.org/wiki/Privacy_Policy)
+
+2. [StaticMap](http://staticmap.openstreetmap.de/). This is used for fetching the map preview.
+
+Using these map services possibly reveals your position and certainly your IP address.
+<hr>
+
+## Links To Other Sites
+
+Fruit Radar contains links to other services that are not directly operated by the developer. If you click on a third party service either in the app, you will be directed to that third party's site. 
+The developer has no control over and assumes no responsibility for the content, privacy policies or practices of any third party sites or services. <hr>
+
+## Transfer Of Data
+
+When the third-party services contained in the app are used, your information, including personal data, may be transferred to computers located outside of your state, province, country or other governmental jurisdiction where the data protection laws may differ than those from your jurisdiction.
+Your consent to this Privacy Policy followed by your submission of such information represents your agreement to that transfer. <hr>
+
+## Children's policy
+The Services of this software does not restrict the utilization of this software by minors or anyone under the age of 13. The concept of this software is intended for general use for all gender, races and age grades. This statement does not hold in localities where policies against the use of this software or any other software in particular by minors exist.  <hr>
+
+## Security of data
+
+The developer values your trust in providing your Personal Information and thus is striving to use commercially acceptable measures of protecting it. However, these measures do not guarantee that your information will not be accessed, disclosed, altered or destroyed by breach of such firewalls and secure server software. By using continuing to use this software and it's services, you acknowledge that you understand and agree to assume these risks. <hr>
+
+## Changes To This Privacy Policy
+
+The developer reserves the right to alter and modify the statements made in this policy at any time. These changes are likely to occur as the development of Fruit Radar software advances, therefore, rendering some statements currently existing invalid. Significant changes will be effective thirty days(30 days) after changes have been made, while minor changes are effective immediately. You'll be notified of these changes via updates on the software. It is advisable to check this policy occasionally for any changes.
```

Comment:
An idea: we can detect the privacy policy changes and and display a dialog.

Reply:
> Significant changes will be effective thirty days(30 days) after changes have been made

If I add a new platform, this will be in the app asap. This is a young app with a rapid development process. I do not think that I would like to tell people that their features need an extra 30 days to get into their devices.

Would it be wise to just notify the user about changes in the app as soon as they are made?
E.g. there is a history of the policy if one is interested in which part changed: https://github.com/niccokunzmann/mundraub-android/commits/master/README.md

Reply:
No I meant the changes made to the privacy policy. If it's a significant change, you can allow maximum 30 days for users to review the new statement and accept the terms in it or not. Then for minor changes to the document, it'll be effective immediately.

Reply:
Then for changes to the app itself, you can notify users immediately via a changelog that'll appear as soon as they update to and launch the latest release of the software.

Reply:
(1) The privacy policy should reflect the app behavior. If I introduce new functionality, the privacy policy should reflect that, even it the new functionality is a major change like adding a new platform. If the software change is introduced, I understand this:

> Significant changes will be effective thirty days(30 days) after changes have been made 

that we need to wait with the feature for 30 days after the policy was changed so that it reflects the feature until the feature is given to the users.

(2) 
> If it's a significant change, you can allow maximum 30 days for users to review the new statement and accept the terms in it or not. 

How will this be done? Is this a pull request open for 30 days?

Reply:
I provide the software under GPLv3 so I do not claim it works. The purpose of the statement in my eyes is e.g. to inform users about what happens to their data, how they can edit the document and how to use or change the app so it works for their privacy needs. So, the review would include the ability to contribute and change the document. But this can be done any time.

Reply:
I think, I understand: Usually, the privacy policy is like a contract between a company holding the data and a user who gives data.

In my case, I do not have any of the data of the users and never will have. It is on their phone and in the place they want to upload it to. I do not think that I need a time period as a safety to change the policy as every version is effective immediately if installed. Changes to the policy can be made any time. There is no warranty, see [the license](https://github.com/niccokunzmann/mundraub-android/blob/master/LICENSE#L44).

My proposal would be that we clarify that changes to the policy are taking effect with every version of the app and may be updated when the app is updated.

What do you think about this?

Reply:
We could do that. After really settling down to understand your point, I found it valid that it's quite awkward for the 30 days waiting period after a feature which leads to additions to this document will be effective. So I can make corrections.

Reply:
You can do that if you like.